extensions/lorem-ipsum/package.json

```diff
@@ -31,18 +32,36 @@
       "data": [
         {
           "value": "clipboard",
-          "title": "Copy to Clipboard"
+          "title": "✂️ Copy to Clipboard"
         },
         {
           "value": "paste",
-          "title": "Paste to Active App"
+          "title": "📋 Paste to Active App"
+        }
+      ],
+      "required": false
+    },
+    {
+      "name": "type",
+      "type": "dropdown",
+      "title": "Ipsum Type",
+      "description": "What type of ipsum would you like to use?",
```

Comment:
I think this could use some clarification. Would it make sense to call it "generator" instead of just "ipsum type"?

Reply:
Along those lines: I, as a user, have no idea what "corporate ipsum" is or why I would prefer one over the other here.